src/test/java/com/wikia/webdriver/common/dataprovider/ads/FandomAdsDataProvider.java

```diff
@@ -50,4 +51,52 @@ private FandomAdsDataProvider() {
         }
     };
   }
+
+    @DataProvider
+    public static Object[][] fandomVuapPage() {
+        return new Object[][]{
+                {
+                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+                        "silicon-valley-perfect-mike-judge",
+                        AdsFandomContent.GPT_TOP_LEADERBOARD,
+                        String.format(
+                                "google_ads_iframe_/5441/wka.fandom/_article/ARTICLE_%s_0",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "google_ads_iframe_/5441/wka.fandom/_article/ARTICLE_%s_0" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)